frontend/src/components/UserMainComponent.vue

```diff
@@ -8,95 +8,62 @@
           <div class="form-group mb-4">
             <input type="text" class="form-control search-user" placeholder="Search User">
           </div>
+          <div class="alert alert-success alert-dismissible fade show" :class="[ alert_button ? '' : 'd-none']" role="alert">
```

Comment:
You can use below format.
Reference: https://v2.vuejs.org/v2/guide/class-and-style.html#Object-Syntax
```suggestion
          <div class="alert alert-success alert-dismissible fade show" :class="{ alert_button: 'd-none'}" role="alert">
```